Src/BootCamp.Chapter/ComputerFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿using BootCamp.Chapter.Computer;
+
+namespace BootCamp.Chapter
+{
+    public abstract class ComputerFactory
+    {
+        protected readonly Body _body;
```

Comment:
Imagine the consequences of doing what you did:
all parts are reference types so essentially you add the same parts to all the computers (same refererence). Therefore- please re-create it.

Reply:
Changed code to install new parts in a new computer instead of the same parts.   My old way seemed more cost effective though.  Thank you very much for guiding me.  It is starting to make sense.